docs/reference/migration/migrate_8_0/security.asciidoc

```diff
@@ -306,6 +306,42 @@ renamed to better reflect its intended use.
 *Impact* +
 Assign users with the `kibana_user` role to the `kibana_admin` role.
 Discontinue use of the `kibana_user` role.
+====
+
+[discrete]
+[[saml-realm-nameid-changes]]
+===== The default value of `nameid_format` setting has been removed.
+
```

Comment:
I'd remove this heading (keep the anchor) and relocate these changes just below the section for `The transport.profiles.*.xpack.security.type setting has been removed.`
```suggestion
```

Reply:
Agreed! I'll implement that change.